app/renderer/components/Inspector/SelectedElement.js

```diff
@@ -126,6 +139,12 @@ class SelectedElement extends Component {
             >
               {t('Clear')}
             </Button>
+            <Tooltip title={t('Copy Attributes to Clipboard')}>
```

Comment:
is this text present in resources?

Reply:
@khanhdodang you'll need to add this to `assets/locales/en/translation.json`

Reply:
Hi @dpgraham , I addressed your comment.